packages/datadog_grpc_interceptor/lib/src/datadog_grpc_interceptor.dart

```diff
@@ -50,10 +60,14 @@ class DatadogGrpcInterceptor extends ClientInterceptor {
     final rumKey = uuid.v1();
     _datadog.rum?.startResourceLoading(
       rumKey,
-      RumHttpMethod.get,
+      RumHttpMethod.unknown,
       fullPath,
       {
-        "grpc.method": method.path,
+        "grpc.method": method.path.split('/').lastWhere(
+              (e) => e.isNotEmpty,
+              orElse: () => method.path,
+            ),
```

Comment:
So, I'm not as well versed on gRPC -- I this generally correct that we should only support sending last portion of the path as the method name?

Reply:
Looking into this -- it does not seem like this is correct in the general case. I'd rather leave the path as the full path, which should include the class and method name of the call.